docs/custom_elements.qmd

```diff
@@ -0,0 +1,194 @@
+# Creating and deploying custom elements
+
+In some cases, you may need to create and deploy custom elements as part of your MLOps workflow using Vetiver. This could be necessary when there is no existing implementation for the type of model you want to deploy, or when you want to implement a current handler in a different way. You may also have a custom element in a known framework, such as a custom column transformer for a scikit-learn model.
+
+In these cases, extra steps will be required to successfully create and deploy a `VetiverModel` object.
+
+# Making a custom model
+
+Vetiver supports basic [scikit-learn](https://scikit-learn.org/), [torch](https://pytorch.org/), [statsmodels](https://www.statsmodels.org/stable/index.html), [xgboost](https://xgboost.readthedocs.io/en/stable/), and [spacy](https://spacy.io/) models. If you need to alter the usage of these models, or deploy a different type of model, you will likely need to create a custom model handler.
+
+To create a custom model handler, you should create a subclass of Vetiver's BaseHandler class. This custom handler should include the following:
+
+- `model_type`:  A static method that declares the type of your model.
```

Comment:
I think this method deserves more explanation. I am confused about what it does and why I need it. Could I just use a string there?

Maybe it would help if the code included a fully executable example that shows defining the custom handler and then using it.

Reply:
Ah, that's a great point. Let me update this to include using the custom handler inside a `VetiverModel` object.